### PR TITLE
Create withdraw modal

### PR DESCRIPTION
**Pull request by JohnDanz**
https://github.com/AugurProject/augur/issues/1340 -- withdraw modal
https://github.com/AugurProject/augur/issues/1369 -- REP faucet

### CODE REVIEW

File: src/modules/modal/modal.styles.less [hunk 1]
```diff
@@ -82,6 +82,119 @@
...
+  flex-flow: row wrap;
+  justify-content: space-between;
+  margin-top: @size-16;
+  padding-top: @size-20;
+
+  > div {
+    margin-bottom: 1rem;
+
+    > span {
+      .text-10;
```
Comment (bthaile) [inline src/modules/modal/modal.styles.less]: Isn't the font size suppose to be 14?

**Comment by bthaile** [comment]
remove the arrows, they should be hidden in all inputs.

Screen Shot 2019-03-19 at 1 00 50 PM

**Comment by bthaile** [comment]
Don't use `address` as the identifier it's not a location, don't want user's location address history to show up. 

Screen Shot 2019-03-19 at 1 02 24 PM

**Comment by bthaile** [comment]
design got mixed up with work-in-progress and finalized design, the withdraw design isn't finalized. leaving this here, waiting for finalized design

**Comment by AugurIntegration** [comment]
This pull request **introduces 1 alert** when merging 92a916c1d02e7c0ff024c138d675fc91637aa2bf into 6ffe4c0abca34870c0242fba6172006a50915fd5 - [view on LGTM.com](https://lgtm.com/projects/g/AugurProject/augur-ui/rev/pr-fb3e31c572cf70e4256e48815919554086c4cfda)

**new alerts:**

* 1 for Useless assignment to local variable

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by bthaile** [comment]
Might be existing bug, I sent MAX initially but had dust left over, which can happen we only estimate gas fee. but clicking on MAX again shouldn't put a negative value in, there should be a insufficient balance message.

Screen Shot 2019-03-20 at 1 27 58 PM

**Comment by bthaile** [comment]
Also the form height jumps when error message shows up, it shouldn't.

Screen Shot 2019-03-20 at 1 31 37 PM

**Comment by AugurIntegration** [comment]
This pull request **introduces 1 alert** when merging b30146cc8e18c2b50bf0bc1fe22a1e059cb7bb56 into 0cce7e773b58e7ff00a70efa8dd279a8ebcf4517 - [view on LGTM.com](https://lgtm.com/projects/g/AugurProject/augur-ui/rev/pr-e6eebbfec63971f5375b03e3e17c90b4e4ccc6a7)

**new alerts:**

* 1 for Useless assignment to local variable

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Pull request merged**